drinfomon.lic

```diff
@@ -857,23 +859,23 @@ while line = script.gets
       if CharSettings['vault_info'].empty?
         respond 'No vault info recorded.'
       else
-        respond "					Your vault is located in #{CharSettings['vault_info']['location']}"
-        respond '					---------------------------------------'
-        respond "					#{CharSettings['vault_info']['contents']}"
+        respond "          Your vault is located in #{CharSettings['vault_info']['location']}"
+        respond "          ---------------------------------------"
+        respond "          #{CharSettings['vault_info']['contents']}"
       end
       respond
     elsif line =~ /(?:In the secure vault you see|You rummage through a secure vault and see)\s(.*)\./
       vault_titles
         .select { |_town, titles| titles.include?(checkroom) }
-        .each do |_town, _titles|
+        .each do |town, _titles|
           contents = Regexp.last_match(1)
           line.scan(/a.*?(?:(\w+ (?:shelf|rack|box|case|tree))) with some stuff (on|in) it/).each do |noun, location|
             fput 'look ' + location + ' ' + noun
             if get =~ /(?:On|In) the \w+ (?:shelf|rack|box|case|tree) you see(.*)\./
               contents += Regexp.last_match(1).sub(/ and(.*)/, ',\1,')
             end
           end
-          CharSettings['vault_info']['location'] = checkarea.slice(/\w+/)
+          CharSettings['vault_info']['location'] = town
```

Comment:
The original logic was grabbing the first word only of a city's name, so for "Fang Cove" it shortened to just "Fang", which to me looked weird. Switched to just use the town the data is for.

Reply:
Will this break for anyone currently using `;vault`?

Reply:
No, it shouldn't. In my investigation `CharSettings['vault_info']['location']` is only used in one other place when outputting your vault info, https://github.com/rpherbig/dr-scripts/blob/a4c3143fd0fa0afa25168674fa1f06aa59f903de/drinfomon.lic#L862